backend/libs/parking/src/commands/handlers/removeParkingCommand.handler.ts

```diff
@@ -16,6 +16,6 @@ export class RemoveParkingCommandHandler implements ICommandHandler<RemoveParkin
 
     //TODO (Larisa) change find one
 
-    //return await this.parkingModel.create(parkingSpace);
+    //return await this.parkingModel.re(parkingSpace);
```

Comment:
Is this correct? What is re?

Reply:
Wasn't aware of this change. But we never use this command so it can be deleted anyway, we only use disableParking

Reply:
Yes